server/node.go

```diff
@@ -221,6 +230,9 @@ func (n *Node) start(rpcServer *rpc.Server, engines []engine.Engine,
 	if err := n.initStores(engines, stopper); err != nil {
 		return err
 	}
+
+	n.startedAt = n.ctx.Clock.Now().WallTime
+	n.storesScanner(stopper)
```

Comment:
I don't know what naming conventions are used for similar scanners, but I think 'startStoresScanner' would be more descriptive.  There's no verb here.